app/views/diagnosis/show.html.haml

```diff
@@ -0,0 +1,9 @@
+%h1= t('.identified_needs', count: @diagnosis.diagnosed_needs.count)
+%p= t('.diagnosis_date', date: I18n.l(@diagnosis.created_at.to_date))
```

Comment:
Add a method in diagnosis model such as :
```
def created_at_date_localized
  I18n.l(created_at.to_date)
end
```
And call it like this `diagnosis.created_at_date_localized`

(Or name it `creation_date_localized`)